tests/extensions/test_extension_python.py

```diff
@@ -31,3 +34,57 @@ def test_grouping_and_choice_failures(test_file, expected_exit_code, expected_me
 
     if expected_exit_code == EXIT_CODE_FAILURE:
         assert "Search failed due to linting errors!" in result.output
+
+
+@pytest.mark.parametrize('test_file,annotations', [
+    (
+        'multiline_simple.pyt',
+        [
+            ('.. pii:', """A long description that
+  spans multiple
+  lines"""),
```

Comment:
I wonder if using `\n`'s in a string on a single line would make this more clear?  Longer results for multi-paragraph might use a combo.  Thoughts?

Reply:
Well I initially used single lines. But the expected results were easier to copy-paste from the test fixtures with multi-line strings. Also, sometimes there were tiny differences in the amount of empty space at the beginning of each line and it was harder to debug. So in this case I preferred multi-line strings.